kola/tests/kubeadm/kubeadm.go

```diff
@@ -71,6 +71,37 @@ var (
 	// testConfig holds params for various kubernetes releases
 	// and the nested params are used to render script templates
 	testConfig = map[string]map[string]interface{}{
+		"v1.25.0": map[string]interface{}{
+			// from https://github.com/flannel-io/flannel/releases
+			"FlannelVersion": "v0.19.1",
+			// from https://github.com/cilium/cilium/releases
+			"CiliumVersion": "1.12.1",
+			// from https://raw.githubusercontent.com/cilium/cilium/master/stable.txt
+			"CiliumCLIVersion": "v1.12.1",
```

Comment:
```suggestion
			// from https://github.com/cilium/cilium-cli/releases/
			"CiliumCLIVersion": "v0.12.1",
```
This is the latest release: https://github.com/cilium/cilium-cli/releases/tag/v0.12.1 for the CLI.

Reply:
Thanks for spotting it. Fixed.